distro/run/assembly/resources/default.yml

```diff
@@ -15,6 +15,8 @@ camunda.bpm:
       allowed-origins: "*"
     rest:
       disable-wadl: false
+    deployment:
```

Comment:
**Property Name Selection Reasoning**:

To be consistent with the docs, the namespace `deployment` was selected to place the new property.

❓ Is this a reasonable choice (find screenshot below of how the namespace is portrayed in the docs )

<details><summary>screenshot</summary>


</details>

If not, agree on the structure of the docs page.